doc/scenarious.md

```diff
@@ -0,0 +1,126 @@
+- Expected Result:
+  A combined table with data from the two related tables is displayed correctly.
+
+---
+
+### Scenario 3: Two Unrelated Tables
+- Description:
+  The user selects two tables that do not have a clear relationship, such as `Customers` and `Products`.
+- Program Logic:
+  - The program informs the user that the tables are unrelated and suggests manually specifying a join condition (via the `ON` clause).
+  - If the user does not specify a join condition, a `CROSS JOIN` is used:
+    ```sql
+    SELECT Customers.CustomerName, Products.ProductName
+    FROM Customers
+    CROSS JOIN Products;
+    ```
+- Expected Result:
+  Either a Cartesian product of the two tables is displayed, or a table based on the manually specified join condition.
+
+---
+
+### Scenario 4: More Than Two Related Tables
+- Description:
+  The user selects three or more related tables, such as `Customers`, `Orders`, and `OrderDetails`.
+- Program Logic:
+  - The program automatically determines the relationships through a chain of foreign keys:
+    - `Customers.CustomerID -> Orders.CustomerID`
+    - `Orders.OrderID -> OrderDetails.OrderID`
+  - If there are ambiguous foreign keys, the program asks the user to select the correct one.
+  - A query is generated:
+    ```sql
+    SELECT Customers.CustomerName, Orders.OrderID, OrderDetails.ProductID, OrderDetails.Quantity
+    FROM Customers
+    INNER JOIN Orders ON Customers.CustomerID = Orders.CustomerID
+    INNER JOIN OrderDetails ON Orders.OrderID = OrderDetails.OrderID;
+    ```
+- Expected Result:
+  A table with combined data from all related tables is displayed correctly.
+
+---
+
+### Scenario 5: Unrelated Tables in the List
+- Description:
+  The user selects several tables, some of which are related while others are not, such as `Customers`, `Orders`, and `Suppliers`.
+- Program Logic:
+  - The program generates an SQL query for the related tables (`Customers` and `Orders`) and provides options for the unrelated tables:
+    - Suggest adding a `CROSS JOIN`.
+    - Allow the user to manually specify a join condition.
+    - Exclude unrelated tables from the query and display a warning.
+- Expected Result:
+  A table with data from the related tables is displayed, while the unrelated tables are either excluded or handled separately.
+
+---
+
+### Scenario 6: More Than Two Unrelated Tables
+- Description:
+  The user selects three or more tables that are not related, such as `Customers`, `Products`, and `Suppliers`.
+- Program Logic:
+  - The program suggests either manually specifying join conditions or using multiple `CROSS JOIN` statements:
+    ```sql
+    SELECT Customers.CustomerName, Products.ProductName, Suppliers.SupplierName
+    FROM Customers
+    CROSS JOIN Products
+    CROSS JOIN Suppliers;
+    ```
+- Expected Result:
+  A table with the Cartesian product of all rows from the selected tables is displayed.
+
+---
+
+### Scenario 7: Tables Without Foreign Keys
```

Comment:
Before you mention exact number, like 2, more than 2, but here no number of tables, makes sense to not focus of number of tables